Makefile

```diff
@@ -1,11 +1,76 @@
 include .bingo/Variables.mk
 
-.DEFAULT_GOAL=textile
+.DEFAULT_GOAL=install
```

Comment:
Should this be `build`? Considering the usual `make` and then `make install` pattern...

Reply:
Not sure. I preserved the behavior that was there before... the `textile` goal ran `go install`. Also my feeing is the traditional `make`, `make install` pattern exists when there is a dependency between the two: first you have to build, then you install the result of build. That isn't the case with the Go commands we're using. Build and install are two separate things.

Reply:
But I really don't have any strong option here. I'll switch it to `build`. Curious though.. when you run `go build ./...`, it seems to just compile the code, but not generate executables anywhere. Is that correct or am I missing something?

Reply:
That's correct, `build` compiles but does not install. Actually, I think you had it right initially. A normal make install would take the result of build and just move the binary... which is not what our install does here (it could move the result of build, but that feels weird given Go has it's own install command)